dev_docs/developer_scripts.md

```diff
@@ -55,8 +55,8 @@ Installs requirements files.
 
 **Usage:** from the `$PROJECT_ROOT/client` directory run `make start-frontend`
 
-NB: the frontend server reads in the desired base_url and dataset name to form the complete url base for API calls. _In
-order to use an arbitrary dataset successfully, the frontend server must be started **after** the backend server_, which
+NB: the frontend server reads in the desired base*url and dataset name to form the complete url base for API calls. \_In
```

Comment:
why was base_url changed to base*url ?

Reply:
I think that was just my auto formatting I only changed the python version on this